src/main/java/org/jvnet/hudson/plugins/repositoryconnector/VersionParameterDefinition.java

```diff
@@ -41,14 +44,15 @@
     private final String groupid;
     private final String repoid;
     private final String artifactid;
+    private final boolean reverseOrder;
```

Comment:
what is the intention of this flag? why would the user wanna have the choise of this? if reverse is better, would we not better change it for everyone?